tests/devices/unit_tests/test_tetramm.py

```diff
@@ -343,7 +343,7 @@ async def test_stage_sets_up_accurate_describe_output(
     assert await tetramm.describe() == {
         TEST_TETRAMM_NAME: {
             "source": "mock+ca://MY-TETRAMM:HDF5:FullFileName_RBV",
-            "shape": [11, 400],
+            "shape": [1, 11, 400],
```

Comment:
FYI @stan-dot and @RJCD-Diamond, this may affect downstream reading of tetramm documents on i18 and i22